bundles/org.openhab.binding.generacmobilelink/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -17,93 +17,115 @@
+			<channel id="signalStrength" typeId="signalStrength"/>
 		</channels>
 		<representation-property>generatorId</representation-property>
 		<config-description-ref uri="thing-type:generacmobilelink:generator"/>
 	</thing-type>
 
-	<channel-type id="connected">
-		<item-type>Switch</item-type>
-		<label>Connected</label>
+	<channel-type id="heroImageUrl">
+		<item-type>String</item-type>
+		<label>Hero Image URL</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="greenLight">
-		<item-type>Switch</item-type>
-		<label>Green Light Status</label>
+	<channel-type id="statusLabel">
+		<item-type>String</item-type>
+		<label>Status Label</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="statusText">
+		<item-type>String</item-type>
+		<label>Status Text</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="activationDate">
+		<item-type>DateTime</item-type>
+		<label>Activation Date</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="yellowLight">
+	<channel-type id="deviceSsid">
+		<item-type>String</item-type>
+		<label>Status Text</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="status">
+		<item-type>Number</item-type>
+		<label>Status</label>
+		<state readOnly="true">
+			<options>
+				<option value="1">Ready</option>
+				<option value="2">Running</option>
+				<option value="3">Exercising</option>
+				<option value="4">Warning</option>
+				<option value="5">Stopped</option>
+				<option value="5">Communication Issue</option>
+				<option value="7">Unknown</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="isConnected">
 		<item-type>Switch</item-type>
-		<label>Yellow Light Status</label>
+		<label>Is Connected</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="redLight">
+	<channel-type id="isConnecting">
 		<item-type>Switch</item-type>
-		<label>Red Light Status</label>
+		<label>Is Connecting</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="blueLight">
+	<channel-type id="showWarning">
 		<item-type>Switch</item-type>
-		<label>Blue Light Status</label>
+		<label>Show Warning</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="statusDate">
-		<item-type>DateTime</item-type>
-		<label>Last Status Date</label>
+	<channel-type id="hasMaintenanceAlert">
+		<item-type>Switch</item-type>
+		<label>Has Maintenance Alert</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="status">
-		<item-type>String</item-type>
-		<label>Status</label>
+	<channel-type id="lastSeen">
+		<item-type>DateTime</item-type>
+		<label>Last Seen</label>
 		<state readOnly="true"/>
 	</channel-type>
-	<channel-type id="currentAlarmDescription">
-		<item-type>String</item-type>
-		<label>Current Alarm Description</label>
+	<channel-type id="connectionTime">
+		<item-type>DateTime</item-type>
+		<label>Connection Time</label>
```

Comment:
I am not 100% sure what the true meaning of the channel is (undocumented API), this shows October of 2022 for me, i'm assuming this means i have had a connection to Generac since then, but I know i have taken down my network for maintenance, and would assumed this would have reset to the last prolonged outage. So i thought its best to leave channels descriptions generic if i'm not certain on their meaning.